src/main/java/com/dnd/favolink/domain/auth/dto/request/SignUpRequest.java

```diff
@@ -0,0 +1,8 @@
+package com.dnd.favolink.domain.auth.dto.request;
+
+import com.dnd.favolink.domain.user.entity.LoginType;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+public record SignUpRequest(@NotBlank String oauthId, @NotBlank String nickname, @NotBlank String profileImg, @NotNull LoginType loginType) {
```

Comment:
```suggestion
public record SignUpRequest(
        @NotBlank String oauthId,
        @NotBlank String nickname,
        @NotBlank String profileImg,
        @NotNull LoginType loginType) {
}
```

개인적으로 이렇게 개행하는 것이 보기 더 좋습니다!

Reply:
오 좋습니다!! 깔끔하니 보기 좋네요!!
수정하도록 하겠습니다 :)